services/vision/vision.go

```diff
@@ -128,6 +128,8 @@ type Service interface {
 
 	// GetObjectPointClouds returns a list of 3D point cloud objects and metadata from the latest 3D camera image using a specified segmenter.
 	GetObjectPointClouds(ctx context.Context, cameraName string, extra map[string]interface{}) ([]*viz.Object, error)
+	// properties
+	GetProperties(ctx context.Context, extra map[string]interface{}) *Properties
```

Comment:
I think you'll want to return an error as well -- in the interest of future proofing, what if you add something to the properties that can easily return an error? You would then have to introduce a breaking change to add the error output